layouts/accounts.html

```diff
@@ -0,0 +1,9 @@
+<h1 id="account-workflow">Accounts</h1>
+
+<p>The Accounts resources are used to manage the lifecycle of patient accounts within the healthcare setting.</p>
+
+<% if @item[:version] == 'dstu2' %>
```

Comment:
We dont support a DSTU2 implementation of this so this can be removed.

Reply:
https://github.com/cerner/fhir.cerner.com/commit/62764511c7f719faa5141a6b7bcbfcc839226e9f

+1